src/Symfony/Component/Uid/Tests/UuidReturnTypeTest.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Symfony\Component\Uid\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Uid\Uuid;
+
+/**
+ * @psalm-type NonEmptyString = non-empty-string
+ */
+class UuidReturnTypeTest extends TestCase
+{
+    private const A_UUID_V4 = 'd6b3345b-8857-9074-a83c-b5988e765d98';
+
+    /**
+     * @psalm-return NonEmptyString
+     */
+    public function testToBinaryReturnsNonEmptyString(): void
```

Comment:
The annotation is wrong. This method does not return anything.

Reply:
@derrabus yes this return type does not make any sense here, I have removed it, thank you.